MANIFEST.in

```diff
@@ -1,2 +1,5 @@
 recursive-include shoop *.jinja *.html *.txt
 recursive-include shoop_tests *.jinja *.html *.txt
+prune shoop/admin/static_src/*/bower_components
+prune shoop/admin/modules/*/bower_components
+prune shoop/*/node_modules
```

Comment:
OK. How do we ensure that "npm run build" has been ran successfully before sdist?